### PR TITLE
Newsletter signup changes

### PR DESCRIPTION
**Pull request by jwade1327**
AW:
Screen Shot 2022-07-19 at 11 41 05 AM
Screen Shot 2022-07-19 at 11 41 16 AM
HCP:
Screen Shot 2022-07-19 at 11 49 04 AM
Screen Shot 2022-07-19 at 11 49 14 AM
OEM:
Screen Shot 2022-07-19 at 11 57 31 AM
Screen Shot 2022-07-19 at 11 57 37 AM
PW:
Screen Shot 2022-07-19 at 12 01 18 PM
Screen Shot 2022-07-19 at 12 01 24 PM
PFW:
Screen Shot 2022-07-19 at 12 05 06 PM
Screen Shot 2022-07-19 at 12 05 13 PM

### CODE REVIEW

**Comment by jwade1327** [comment]
[PFW: Newsletter Signup Info](https://trello.com/c/g8IIaZ8V/345-pfw-newsletter-signup-info)

**Comment by jwade1327** [comment]
[PW: Newsletter Signup Info](https://trello.com/c/cI5jwQDc/344-pw-newsletter-signup-info)

**Comment by jwade1327** [comment]
[OEM: Newsletter Signup Info](https://trello.com/c/RLM4VRu9/346-oem-newsletter-signup-info)

**Comment by jwade1327** [comment]
[HCP: Newsletter Signup Info](https://trello.com/c/vgksYrC2/347-hcp-newsletter-signup-info)

**Review by B77Mills — APPROVED** [review]

File: packages/global/components/document.marko [hunk 1]
```diff
@@ -10,6 +10,7 @@ $ const {
 } = out.global;
 
 $ const continueLable = i18n("Sign Me Up!");
```
Comment (brandonbk) [inline packages/global/components/document.marko:12]: `continueLable` is incorrectly spelled; is it wrong in the low-level component as well?

File: packages/global/components/layouts/content/default.marko [hunk 2]
```diff
@@ -14,6 +14,7 @@ $ const displaySocialShare = ["contact"].includes(type) ? false : true;
 $ const displayComments = false; // ["contact"].includes(type) ? false : true;
 $ const displayNewsletterSignup = ["contact"].includes(type) ? false : true;
 $ const continueLable = i18n("Sign Me Up!");
```
Comment (brandonbk) [inline packages/global/components/layouts/content/default.marko:16]: `continueLable` spelled wrong here as well

**Review by brandonbk — CHANGES_REQUESTED** [review]

**Pull request merged**